<?xml version="1.0"?>

<launch>
    <arg name="hostname_front" default="192.168.13.121"/>
    <arg name="cloud_topic_front" default="cloud_front"/>
    <arg name="frame_id_front" default="sickF"/>
    <node name="sick_tim_7xx_front" pkg="sick_scan" type="sick_generic_caller" respawn="false" output="screen">
        <param name="scanner_type" type="string" value="sick_tim_7xxS"/>
        <param name="min_ang" type="double" value="-2.25"/><!-- -2.35619449-->
        <param name="max_ang" type="double" value="2.25"/><!-- 2.35619449-->
        <param name="use_binary_protocol" type="bool" value="true"/>
        <param name="range_max" type="double" value="100.0"/>
        <param name="intensity" type="bool" value="True"/>
        <param name="hostname" type="string" value="$(arg hostname_front)"/>
        <param name="cloud_topic" type="string" value="$(arg cloud_topic_front)"/>
        <param name="frame_id" type="str" value="$(arg frame_id_front)"/>
        <param name="port" type="string" value="2112"/>
        <param name="timelimit" type="int" value="5"/>
				<remap from="scan" to="scanF"/>
    </node>

    <arg name="hostname_rear" default="192.168.13.122"/>
    <arg name="cloud_topic_rear" default="cloud_rear"/>
    <arg name="frame_id_rear" default="sickR"/>
    <node name="sick_tim_7xx_rear" pkg="sick_scan" type="sick_generic_caller" respawn="false" output="screen">
        <param name="scanner_type" type="string" value="sick_tim_7xxS"/>
        <param name="min_ang" type="double" value="-2.25"/>
        <param name="max_ang" type="double" value="2.25"/>
        <param name="use_binary_protocol" type="bool" value="true"/>
        <param name="range_max" type="double" value="100.0"/>
        <param name="intensity" type="bool" value="True"/>
        <param name="hostname" type="string" value="$(arg hostname_rear)"/>
        <param name="cloud_topic" type="string" value="$(arg cloud_topic_rear)"/>
        <param name="frame_id" type="str" value="$(arg frame_id_rear)"/>
        <param name="port" type="string" value="2112"/>
        <param name="timelimit" type="int" value="5"/>
				<remap from="scan" to="scanR"/>
    </node>
</launch>

